$NetBSD: patch-Makefile.conf,v 1.2 2014/10/09 21:18:05 jaapb Exp $

Have BINDIR be determined from package Makefile, do not call git
--- Makefile.conf.orig	2014-09-30 14:40:37.000000000 +0000
+++ Makefile.conf
@@ -1,6 +1,6 @@
 
 # Where binaries are installed:
-BINDIR := `dirname  \`which ocamlc\``
+#BINDIR := `dirname  \`which ocamlc\``
 
 ####
 
@@ -47,4 +47,4 @@ NATDYNLINK ?= $(shell if [ -f `ocamlc -w
 
 METAOCAML ?= 0
 
-VERSION_GIT := $(shell git log -n1 --pretty=format:%h 2> /dev/null)
+#VERSION_GIT := $(shell git log -n1 --pretty=format:%h 2> /dev/null)
